documentation/design_docs/MIP_Engine_Interface.yml

```diff
@@ -49,269 +49,202 @@ paths:
                 $ref: '#/components/schemas/AlgorithmResult'
         '400':
           description: 'algorithm does not exist or badly formatted input'
-      
+
 
 components:
   schemas:
     Algorithm:
       description: All the details and properties that specify an algorithm.
       example: {
```

Comment:
put the examples in a seperate section and &ref them in the specification. It makes it very unreadable to have a 110 line example in the middle of the resource specification

components:
 ...
 schemas:
     Algorithm:
       description: All the details and properties that specify an algorithm.
       examples:
         example1:
           $ref: #components/examples/algorithm
         ...

examples:
   algorithm:
       value:{
           "name": "ALGORITHMNAME",
           "desc": "This is an example algorithm.",
           "desc": "This is an example algorithm.",
           ...
}